bitnami/kafka/templates/broker/statefulset.yaml

```
@@ -152,7 +152,7 @@ spec:
             - name: KAFKA_KRAFT_CLUSTER_ID
               valueFrom:
                 secretKeyRef:
-                  name: {{ printf "%s-kraft-cluster-id" (include "common.names.fullname" .) }}
+                  name: {{ ternary (printf "%s-kraft-cluster-id" (include "common.names.fullname" .)) .Values.kraft.existingClusterIdSecret  (not .Values.kraft.existingClusterIdSecret) }}
```

Searching over a negated value in a ternary seems a little tricky. What about...

```suggestion
                  name: {{ default (printf "%s-kraft-cluster-id" (include "common.names.fullname" .)) .Values.kraft.existingClusterIdSecret }}
```